### PR TITLE
Nextflow combined files check

### PR DESCRIPTION
**Pull request by NickSwainston**

### CODE REVIEW

File: scripts/check_known_pulsars.py [hunk 1]
```diff
@@ -702,7 +702,7 @@ def beamform_and_fold(obsid, DI_dir, cal_obs, args, psrbeg, psrend,
         find_beg_end(args.obsid, base_path=comp_config['base_product_dir'])
 
     #Perform data checks
-    dur = end-beg
+    dur = end-beg + 1
```
Comment (keegansmith21) [inline scripts/check_known_pulsars.py:705]: Thanks, my bad

**Review by keegansmith21 — COMMENTED** [review]
I'm no good at nextflow but is there a way to spit out what files are missing?

File: scripts/check_known_pulsars.py (hunk 1 above)
Comment (NickSwainston) [reply scripts/check_known_pulsars.py:705]: all g I got ya back

**Comment by NickSwainston** [comment]
It's running our python scripts so if you look in the .command.err it will output what is missing

**Pull request merged**